<!DOCTYPE HTML>
<!--
	Minimaxing by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title> Software - Tang Lab </title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
	</head>
	<body>
		<div id="page-wrapper">
			<div id="header-wrapper">
				<div class="container">
					<div class="row">
						<div class="12u">

							<header id="header">
								<h1><a href="#" id="logo"><font style="text-transform: none;"> Tang Lab </font></a></h1>
								<nav id="nav">
									<a href="index.html"><font style="text-transform: none;"> Home </font></a>
									<a href="research.html"><font style="text-transform: none;"> Research </font></a>
									<a href="publications.html"><font style="text-transform: none;"> Publications </font></a>
									<a href="software.html" class="current-page-item"><font style="text-transform: none;"> Software </font></a>
									<a href="people.html"><font style="text-transform: none;"> People </font></a>
								</nav>
							</header>

						</div>
					</div>
				</div>
			</div>
			<div id="main">
				<div class="container">
					<div class="row main-row">
						<div class="12u">

						<h2>Software</h2>
						
<p><strong><font size="5">miLineage: An R package to perform association tests for microbial lineages on a taxonomic tree </font></strong></p>
<p>miLineage package has functions that implement the distribution-free association tests for microbiome data described in Tang et al. (2016). These functions allow users to: (a) perform robust association tests at any taxonomic level for microbiome data that exhibit arbitrary inter-taxa correlation; (b) localize the covariate-associated lineages on the taxonomic tree; and (c) assess the overall association of the microbial community. Our framework does not make any distributional assumptions on the microbiome data; it allows for the adjustment of confounding variables and accommodates excessive zero observations; and it incorporates taxonomic information. Simulation studies and real data analyses demonstrate substantial power gain over competing methods. </p>
<p><font size="5"><strong>References:</strong></font><a href="https://academic.oup.com//bioinformatics/article/33/9/1278/2729907/A-general-framework-for-association-analysis-of?guestAccessKey=6c701407-b46c-49b1-a4d3-852545eeeead">Tang ZZ <i>et al.</i> 2016b </a>.</p>
<footer class="controls">
<a href="software/miLineage_v1.0" class="button"><font style="text-transform: none;"> Download miLineage </font></a>
</footer>

<p>miProfile: A C program for analyzing&nbsp;microbial community composition</p>

<p>https://medschool.vanderbilt.edu/tang-lab/software/miProfile</p>

<p>citation:&nbsp;Tang ZZ,&nbsp;Chen G,&nbsp;Alekseyenko&nbsp;AV&nbsp;(2016). PERMANOVA-S:&nbsp;Association test for microbial community composition that accommodates confounders and multiple distances,&nbsp;Bioinformatics, btw311.</p>

<p>&nbsp;</p>

<p>PreMeta: A C program to&nbsp;facilitate the exchange of information between four software packages for meta-analysis of rare-variant associations: MASS, RAREMETAL, MetaSKAT, and seqMeta.</p>

<p>http://dlin.web.unc.edu/software/premeta/</p>

<p>&nbsp;</p>

<p>MASS: A C program for the meta-analysis of sequencing studies.</p>

<p>http://dlin.web.unc.edu/software/mass/</p>

<p>&nbsp;</p>

<p>SCORE-SeqTDS: A C program for analyzing primary and secondary traits in sequencing studies under trait- dependent sampling.</p>

<p>http://dlin.web.unc.edu/software/score-seqtds/</p>

<p>&nbsp;</p>

<p>SCORE-Seq: A C program which implements score statistics for detecting disease associations with rare variants in sequencing studies.</p>

<p>http://dlin.web.unc.edu/software/score-seq/</p>

<p>&nbsp;</p>

<p>genoCN: An R package for identifying copy number states and genotype calls using the SNP arrays data.</p>

<p>https://www.bioconductor.org/packages/release/bioc/html/genoCN.html</p>

							

						</div>
					</div>
				</div>
			</div>
			<div id="footer-wrapper">
				<div class="container">
					<h2>Contact Us</h2>
					
					<div class="row">
						
						<div class="8u 12u(mobile)">

							<section>
								<p> Mailing address: 330 N Orchard St, Madison, WI 53715, United States </p>
								<p>	Office location: WID 3268 </p>
							</section>

						</div>
						<div class="4u 12u(mobile)">

							<section>
								<p> Email: tang@biostat.wisc.edu </p>
								<p>	Office phone: (608) 316-4442 </p>
							</section>

						</div>

					</div>
					<div class="row">
						<div class="12u">

							<div id="copyright">
								&copy; ZhengZheng Tang 2017. All rights reserved. | Design: <a href="http://html5up.net">HTML5 UP</a> | Last updated July 1, 2017
							</div>

						</div>
					</div>
				</div>
			</div>
		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/skel-viewport.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>

	</body>
</html>
